app/jobs/update_entity_source.rb

```diff
@@ -3,8 +3,8 @@
 require 'metadata/schema'
 require 'net/http'
 
-class UpdateEntitySource
-  include SetSAMLTypeFromXML
+class UpdateEntitySource <
```

Comment:
This doesn't look right 🤔. Was it intentional?

```suggestion
class UpdateEntitySource
```

Reply:
Ah that was me playing around with the `includes` and stuff, was not intended to commit those changes.